helm/agent/templates/ingress.yaml

```diff
@@ -61,7 +61,7 @@ spec:
           {{- end }}
     {{- end }}
 ---
-{{- $grpcPort := .Values.service.ports.grpc -}}
+{{ $grpcPort := .Values.service.ports.grpc -}}
```

Comment:
I found another project had the[ same problem](https://github.com/vidispine/hull/issues/186#issuecomment-1294856635)